worldwind/src/main/java/gov/nasa/worldwind/shape/TextAttributes.java

```diff
@@ -104,7 +104,7 @@ public int hashCode() {
         result = 31 * result + (this.textSize != +0.0f ? Float.floatToIntBits(this.textSize) : 0);
         result = 31 * result + (this.typeface != null ? this.typeface.hashCode() : 0);
         result = 31 * result + (this.enableOutline ? 1 : 0);
-        result = 31 * result + (this.outlineColor != null ? this.outlineColor.hashCode() : 0);
+        result = 31 * result + this.outlineColor.hashCode();
```

Comment:
Nice catch. TextAttributes guarantees that textColor and outlineColor are non-null, correct?

Reply:
@pdavidc , yes, the setter is protected by a null check. In the extreme, the field isn't private, so could be modified to null. I assumed that would be beyond the scope of our check, right?

Reply:
Yes, your assessment is correct. Android adopted the principle that we need not protect ourselves from poorly written subclasses.